src/goto-cc/goto_cc_cmdline.cpp

```diff
@@ -24,7 +24,14 @@ Date:   April 2010
 goto_cc_cmdlinet::~goto_cc_cmdlinet()
 {
   if(!stdin_file.empty())
-    remove(stdin_file.c_str());
+  {
+    int result=remove(stdin_file.c_str());
+    if(result!=0)
+    {
+      // Let's print the error to stderr instead of ignoring it completely
+      std::cerr << "Remove failed: " << std::strerror(errno) << std::endl;
```

Comment:
Don't use `std::endl`; you might actually do `perror("Remove failed");` instead of the std::cerr play.